packages/panels/src/Commands/MakePageCommand.php

```diff
@@ -47,8 +48,34 @@ public function handle(): int
         $resourceClass = null;
         $resourcePage = null;
 
-        $resourceInput = $this->option('resource') ?? text(
-            label: 'What is the resource you would like to create this in?',
+        $panel = $this->option('panel');
+
+        if ($panel) {
+            $panel = Filament::getPanel($panel);
+        }
+
+        if (! $panel) {
+            $panels = Filament::getPanels();
+
+            /** @var Panel $panel */
+            $panel = (count($panels) > 1) ? $panels[select(
+                label: 'Which panel would you like to create this in?',
+                options: array_map(
+                    fn (Panel $panel): string => $panel->getId(),
+                    $panels,
+                ),
+                default: Filament::getDefaultPanel()->getId()
+            )] : Arr::first($panels);
+        }
+
+        $resourceInput = $this->option('resource') ?? suggest(
+            label: 'Which resource would you like to create this in?',
+            options: collect($panel->getResources())
+                ->map(
+                    fn ($resource): string => str($resource)
+                        ->afterLast('\\')
```

Comment:
```suggestion
                        ->afterLast('Resources\\')
```

to handle nested directories?

Reply:
Hmm. What would a nested directory path look like?

Reply:
App\Filament\Resources\Blog\PostResource